src/Microsoft.Azure.WebJobs.Host/Loggers/FunctionInstanceLogger.cs

```diff
@@ -47,7 +48,12 @@ public Task LogFunctionCompletedAsync(FunctionCompletedMessage message, Cancella
         private void Log(LogLevel level, FunctionDescriptor descriptor, Guid functionId, string message, Exception exception = null)
         {
             ILogger logger = _loggerFactory?.CreateLogger(LogCategories.CreateFunctionCategory(descriptor.LogName));
-            logger?.Log(level, 0, message, exception, (s, e) => s);
+            var logData = new Dictionary<string, object>
+            {
+                [LogConstants.LogSummaryKey] = message,
```

Comment:
Didn't see an answer to my previous question: "Why put the message in the properties? I haven't seen us do this anywhere else in our logging code. I thought the log data was just for additional metadata."

Reply:
Answered this one below:
Earlier logstate was just a string message, now including functionInviocationId in the log state so that systemlogger in the functions runtime can access it.
@brettsam - Let me know if there is a better way to do this.

Reply:
Will add a constant for Summary. Earlier logstate was just a string message, now including functionInviocationId in the log state so that systemlogger in the functions runtime can access it.
@brettsam - Let me know if there is a better way to do this.